apple/watchos.doc.bzl

```diff
@@ -0,0 +1,22 @@
+"""Bazel rules for creating watchOS applications and bundles."""
```

Comment:
Can this be an H1 header?

Reply:
Sure, pushed another commit changing all the module docstrings to start with H1 to keep the generated pages consistent.

https://github.com/bazelbuild/rules_apple/blob/b6e4595f4eb7be613e3a267d395d808f85009b40/doc/rules-watchos.md
is the preview